### PR TITLE
Fix/eth1notifier tests

### LINKED ISSUE #167: Test suite unhandled promise rejection
**Describe the bug**

There is an unhandled promise rejection within the test sutie for `intToBytes` that does not cause the CI to fail.

**Expected behavior**

The CI should fail, and the test should probably also fail.

**Steps to Reproduce**

1. `yarn install`
2. `yarn test`

**Screenshots**

```
  intToBytes
    ✓ should correctly serialize number to bytes length 1
    ✓ should correctly serialize BN to bytes length 1
    ✓ should correctly serialize number to bytes length 2
    ✓ should correctly serialize BN to bytes length 2
    ✓ should correctly serialize number to bytes length 3
    ✓ should correctly serialize BN to bytes length 3
    ✓ should correctly serialize number to bytes length 4
    ✓ should correctly serialize BN to bytes length 4
    ✓ should correctly serialize number to bytes length 8
    ✓ should correctly serialize BN to bytes length 8
    ✓ should correctly serialize number to bytes length 32
    ✓ should correctly serialize BN to bytes length 32
    ✓ should correctly serialize number to bytes length 48
    ✓ should correctly serialize BN to bytes length 48
(node:67078) UnhandledPromiseRejectionWarning: Error: network does support ENS (operation="ENS", network="unknown", version=4.0.27)
    at Object.throwError (/Users/gregmarkou/Jobs/code/github.com/ChainSafe/lodestar/node_modules/ethers/errors.js:76:17)
    at /Users/gregmarkou/Jobs/code/github.com/ChainSafe/lodestar/node_modules/ethers/providers/base-provider.js:1007:24
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:67078) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:67078) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Desktop (please complete the following information):**
 - OS:  OSX
 - Branch:  Master

### PR DESCRIPTION
**Pull request by mpetrunic**
Should resolve #167 

I've added a check to verify if contract exists and it's address is valid and therefore it removed unhandled promise rejection.

### CODE REVIEW

File: src/node/index.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ interface Service {
 interface BeaconNodeCtx {
   chain?: object;
   db?: object;
-  eth1?: object;
+  eth1?: any;
```
Comment (GregTheGreek) [inline src/node/index.ts:25]: Can we use an alternative to `any`?

**Review by GregTheGreek — CHANGES_REQUESTED** [review]
One question otherwise LGTM

**Review by GregTheGreek — APPROVED** [review]

File: src/node/index.ts (hunk 1 above)
Comment (mpetrunic) [reply src/node/index.ts:25]: Probably should have put interface there. I'm not sure why objects are type of each config haha
Will try to put Eth1Options interface and check if it breaks anything

Comment (mpetrunic) [reply src/node/index.ts:25]: Nope, doesn't work, since Eth1Notifier requires all props, and all of beacon opts should be optional so it has to be any :/

Comment (GregTheGreek) [reply src/node/index.ts:25]: ahh too bad ... no worries

Comment (GregTheGreek) [reply src/node/index.ts:25]: You've got my approval

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**